pkg/tarmak/instance_pool/instance_pool.go

```diff
@@ -206,45 +206,71 @@ func (n *InstancePool) ValidateAllowCIDRs() (result error) {
 
 func (n *InstancePool) Labels() (string, error) {
 	var labels []string
+
+	err := n.validLabels()
+	if err != nil {
+		return "", err
+	}
+
+	for _, label := range n.conf.Labels {
+		labels = append(labels, fmt.Sprintf("  %s: \"%s\"", label.Key, label.Value))
+	}
+
+	return strings.Join(labels, "\n"), nil
+}
+
+func (n *InstancePool) validLabels() error {
```

Comment:
Can we use that instead: https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/validation/validation.go#L66